modules/installation-ibm-z-user-infra-machines-iso.adoc

```diff
@@ -149,7 +149,7 @@ zfcp.allow_lun_scan=0
 Write all options in the parameter file as a single line and make sure you have no newline characters.
 
 ifdef::ibm-z[]
-. Transfer the initramfs, kernel, parameter files, and {op-system} images to z/VM, for example with FTP. For details about how to transfer the files with FTP and boot from the virtual reader, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/installation_guide/sect-installing-zvm-s390[Installing under Z/VM].
+. Transfer the initramfs, kernel, parameter files, and {op-system} images to z/VM, for example with FTP. For details about how to transfer the files with FTP and boot from the virtual reader, see link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html-single/interactively_installing_rhel_over_the_network/index#using-the-zvm-reader_installing-under-z-vm[Booting the installation on {ibm-z-name} to install {op-system-base} in z/VM].
```

Comment:
Usually the link metadata would match the link target heading, which is Booting the RHEL installation by using the z/VM Reader.

Reply:
ACK.

Reply:
@dfitzmau Thanks for spotting this I just realized the toc has a real strange behaviour. When I click on 12.4 once I get the 12.4.1 link, with a second click on 12.4 I get the correct one 
![image](https://github.com/user-attachments/assets/5824078e-4b7e-4054-8bc1-da5db1a2b1cf)

Reply:
Is this a D.R.C issue? Andrea raised https://redhat-internal.slack.com/archives/C073SBBME1M/p1726663903236999 in the forum-docs-redhat Slack channel. The TOC on D.R.C is troublesome at times.